src/plugins/mer/meremulatordevice.cpp

```diff
@@ -154,6 +154,8 @@ private slots:
             setValue(0);
             setLabelText(tr("Error occured"));
             setCancelButtonText(tr("Close"));
+            close();
```

Comment:
Could have just used [[fallthrough]]

Reply:
Or rather Q_FALLTHROUGH, but I found it more clear not to do so in this case.